experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormFeatureValue/FeatureValueEditor.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Form from "react-bootstrap/Form";
+import { useController, useFormContext } from "react-hook-form";
+import RichFeatureValueEditor from "src/components/PageEditBranches/FormBranches/FormFeatureValue/RichFeatureValueEditor";
+import { useCommonNestedForm } from "src/hooks";
+import { getConfig_nimbusConfig_allFeatureConfigs } from "src/types/getConfig";
+
+export interface FeatureValueEditorProps {
+  featureConfig: getConfig_nimbusConfig_allFeatureConfigs;
+  formControlAttrs: ReturnType<
+    ReturnType<typeof useCommonNestedForm>["formControlAttrs"]
+  >;
+}
+
+let FeatureValueEditor: (props: FeatureValueEditorProps) => JSX.Element;
+
+if (process.env.JEST_WORKER_ID !== undefined) {
+  FeatureValueEditor = function FeatureValueEditor({
+    formControlAttrs,
+  }: FeatureValueEditorProps) {
+    return <Form.Control {...{ ...formControlAttrs }} as="textarea" rows={4} />;
+  };
```

Comment:
This is falling back to the existing input for the unit test issue yeah?

Reply:
Yes